.env.example

```diff
@@ -1,4 +0,0 @@
-SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN='<Access token provided by mapbox to use the service>'
```

Comment:
hey is there any reason you deleted this? Last i checked we added this so that devs would have a better idea of what environment variables they needed.

Reply:
oh, good call!
No reason for deleting this, figured i was supposed to once I replaced with the actual `.env` file.
I'll bring it back